tpe/plugin/src/SimulationFeatures.cc

```diff
@@ -58,6 +62,51 @@ void SimulationFeatures::WorldForwardStep(
     }
   }
   world->Step();
+  this->WriteRequiredData(_h);
+  this->Write(_h.Get<JointPositions>());
+}
+
+void SimulationFeatures::Write(JointPositions &/*_positions*/) const
+{
+  // TODO(adlarkin) implement this? Is this needed?
+}
```

Comment:
Similar to my comment about this method in the DART implementation, I don't believe that we need this method since we only need to know which links have moved (https://github.com/ignitionrobotics/ign-physics/issues/219). I can remove this method as well if this is true.

Reply:
removed in https://github.com/ignitionrobotics/ign-physics/pull/223/commits/ad12778a0af2f312222f108db4bd90ebf22502cb